types/carbon-components-react/lib/components/ToggleSmall/ToggleSmall.d.ts

```diff
@@ -6,7 +6,7 @@ interface InheritedProps extends
     Omit<ReactInputAttr, ExcludedAttributes>,
     RequiresIdProps
 {
-    ariaLabel: NonNullable<React.AriaAttributes["aria-label"]>,
+    ["aria-Label"]: NonNullable<React.AriaAttributes["aria-label"]>,
```

Comment:
Should this be `aria-label`(lowercase L) or does it matter?

Reply:
yes you're correct. I'll push my changes